server/build/gzip.js

```diff
@@ -8,7 +8,8 @@ export default async function gzipAssets (dir) {
 
   const coreAssets = [
     path.join(nextDir, 'commons.js'),
-    path.join(nextDir, 'main.js')
+    path.join(nextDir, 'main.js'),
+    path.join(nextDir, 'static-bundle.js')
```

Comment:
Technically, these three are serving three different purposes. It's possible to merge to all.
But with HTTP2, having multiple files is better. 
(Even without that, browsers starts to download them in parallel)

So, it's better to have many files rather one.